plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3284,16 +3284,22 @@ public void testVoidTransform()
         assertUpdate("INSERT INTO test_void_transform " + values, 7);
         assertQuery("SELECT * FROM test_void_transform", values);
 
-        assertQuery("SELECT COUNT(*) FROM \"test_void_transform$partitions\"", "SELECT 1");
```

Comment:
what was wrong with this?

Reply:
Replaced with the assert below where we select all the rows.